frontend/i18n/en-US.json

```diff
@@ -128,6 +128,7 @@
   "components.card-danger-zone.username-placeholder": "Enter your username",
   "components.card-discussion-input.comment": "Comment",
   "components.card-discussion-input.leave-comment": "Leave a comment",
```

Comment:
Can we change this to `Leave a public comment` to match the designs? I can also do this myself 😊